obs-package-status/README.md

```diff
@@ -0,0 +1,65 @@
+# OBS Package Status Scanner
+This script shows if the packages of OBS (Open Build Service) are broken or not. The status gets then reported to a Trello card.
+
+## Configuration
+This bot is configured via environment variables. Either set them however you set them
+or copy the `dotenv.example` file to `.env` and the script picks them up from this file.
```

Comment:
`dotenv.example` should be renamed to `.env.example`

Reply:
Not sure a 'hidden' file in the repo this is too obvious...